mobile/src/main/res/layout/decklist_card_row.xml

```diff
@@ -1,63 +1,79 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:orientation="vertical"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:layout_height="wrap_content"
+    android:background="?attr/selectable_row">
 
     <LinearLayout
-        android:id="@+id/decklistSeparator"
-        android:orientation="horizontal"
+        android:id="@+id/card_row_full"
+        android:orientation="vertical"
         android:layout_width="match_parent"
         android:layout_height="wrap_content">
 
-        <TextView
-            tools:text="Header"
-            android:id="@+id/decklistHeaderType"
-            android:layout_height="wrap_content"
-            android:layout_width="wrap_content"
-            style="?android:attr/listSeparatorTextViewStyle" />
+        <LinearLayout
+            android:id="@+id/decklistSeparator"
+            android:orientation="horizontal"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <TextView
+                tools:text="Header"
+                android:id="@+id/decklistHeaderType"
+                android:layout_height="wrap_content"
+                android:layout_width="wrap_content"
+                style="?android:attr/listSeparatorTextViewStyle" />
+
+            <TextView
+                tools:text="(##)"
+                android:id="@+id/decklistHeaderNumber"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                style="?android:attr/listSeparatorTextViewStyle" />
 
-        <TextView
-            tools:text="(##)"
-            android:id="@+id/decklistHeaderNumber"
+        </LinearLayout>
+
+        <LinearLayout
+            android:id="@+id/card_row"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            style="?android:attr/listSeparatorTextViewStyle" />
-    </LinearLayout>
+            android:layout_height="48sp"
```

Comment:
This should be dp, not sp (which is generally reserved for text sizes). Upon further inspection, it's just wrapping a bunch of text, so I guess it's fine as-is.